### PR TITLE
Add dustmaps requirement

### PR DESCRIPTION
**Pull request by SimonKrughoff**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Review by ktlim — APPROVED** [review]
Could you place this in the correct location in alphabetical order?  It makes it slightly easier to find things.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% set name = "rubin-env" %}
-{% set version = "0.2.1" %}
+{% set version = "0.3.1" %}
```
Comment (brianv0) [inline recipe/meta.yaml:2]: If bumping minor version, can you start at 0.3.0?

Comment (SimonKrughoff) [reply recipe/meta.yaml:2]: Of course.

**Comment by brianv0** [comment]
looks good - I can merge

**Pull request merged**